documentation/src/main/asciidoc/topics/con_cache_encoding_clients_text.adoc

```diff
@@ -0,0 +1,29 @@
+[id='cache-encoding-clients-text_{context}']
+= Text formats and remote clients
+Text-based encoding is human-readable content like plain text or HTML.
+The classic "Hello World" example entry could be represented as follows:
+
+[source,text,options="nowrap",subs=attributes+]
+----
+key=hello
+value=world
+----
+
+[discrete]
+== Hot Rod clients
+
+{brandname} can handle `String` and `byte[]` types natively.
+If you configure cache encoding with the `text/plain` media type then you can use Hot Rod clients without any marshaller configuration.
```

Comment:
@danberindei Doesn't that match what is in the docs here? If the remote cache is encoded as `text/plain` then you can read/write `String` and `byte[]` along with some other types without the need to configure a marshaller because Infinispan does the transcoding with ProtoStream.

Reply:
I've reworked this. Let's see.

Reply:
@danberindei Please take a look at the updates here: https://github.com/infinispan/infinispan/pull/9106/files#diff-22728ae7120518b81bae1f844cc040a5a8c0bba2630b87859e61cff8b18be40bR9